### PR TITLE
Fixes for events

### PR DESCRIPTION
**Pull request by Amaimon208**

### CODE REVIEW

File: app/src/main/java/com/calendar/activity/CreateEventActivity.java [hunk 1]
```diff
@@ -70,23 +72,36 @@ public void showAdvancedOptions(View view) {
             annualCheckBox.setVisibility(View.VISIBLE);
             reminderCheckBox.setVisibility(View.VISIBLE);
             freeCheckBox.setVisibility(View.VISIBLE);
+            showReminderTimeCheck();
```
Comment (P1200) [inline app/src/main/java/com/calendar/activity/CreateEventActivity.java:75]: why not use `showReminderClockCheck()`?

File: app/src/main/java/com/calendar/activity/CreateEventActivity.java [hunk 2]
```diff
@@ -70,23 +72,36 @@ public void showAdvancedOptions(View view) {
...
             reminderTimeCheckBox.setVisibility(View.INVISIBLE);
         }
     }
 
     public void showReminderClock(View view) {
-        if (reminderTimeCheckBox.isChecked()) {
+        showReminderClockCheck();
+    }
+
+    public void showReminderClockCheck() {
```
Comment (P1200) [inline app/src/main/java/com/calendar/activity/CreateEventActivity.java:103]: Can be private?

File: app/src/main/java/com/calendar/activity/CreateEventActivity.java [hunk 3]
```diff
@@ -70,23 +72,36 @@ public void showAdvancedOptions(View view) {
...
         } else {
             annualCheckBox.setVisibility(View.INVISIBLE);
             reminderCheckBox.setVisibility(View.INVISIBLE);
             freeCheckBox.setVisibility(View.INVISIBLE);
+            reminderTimeCheckBox.setVisibility(View.INVISIBLE);
+            timePicker.setVisibility(View.INVISIBLE);
         }
     }
 
     public void showReminderTimeCheckBox(View view) {
```
Comment (P1200) [inline app/src/main/java/com/calendar/activity/CreateEventActivity.java:86]: Can be private?

File: app/src/main/java/com/calendar/activity/CreateEventActivity.java [hunk 4]
```diff
@@ -70,23 +72,36 @@ public void showAdvancedOptions(View view) {
...
-        if (reminderCheckBox.isChecked()) {
+        showReminderTimeCheck();
+        showReminderClockCheck();
+    }
+
+    private void showReminderTimeCheck(){
+        if (advancedSettingsCheckBox.isChecked() && reminderCheckBox.isChecked()) {
```
Comment (P1200) [inline app/src/main/java/com/calendar/activity/CreateEventActivity.java:92]: Mbe move statement inside `if` to another function

File: app/src/main/java/com/calendar/activity/CreateEventActivity.java [hunk 5]
```diff
@@ -70,23 +72,36 @@ public void showAdvancedOptions(View view) {
...
         }
     }
 
     public void showReminderClock(View view) {
-        if (reminderTimeCheckBox.isChecked()) {
+        showReminderClockCheck();
+    }
+
+    public void showReminderClockCheck() {
+        if (advancedSettingsCheckBox.isChecked()  && reminderCheckBox.isChecked() &&  reminderTimeCheckBox.isChecked()) {
```
Comment (P1200) [inline app/src/main/java/com/calendar/activity/CreateEventActivity.java:104]: Mbe move statement inside `if` to another function.

File: app/src/main/java/com/calendar/activity/CreateEventActivity.java (hunk 1 above)
Comment (Amaimon208) [reply app/src/main/java/com/calendar/activity/CreateEventActivity.java:75]: I didn't want to pass view to the function

File: app/src/main/java/com/calendar/activity/CreateEventActivity.java (hunk 3 above)
Comment (Amaimon208) [reply app/src/main/java/com/calendar/activity/CreateEventActivity.java:86]: No it can, becouse it is connected to button

File: app/build.gradle [hunk 6]
```diff
@@ -58,4 +57,13 @@ dependencies {
...
     implementation 'com.google.android.gms:play-services-ads:22.0.0'
+
+    androidTestImplementation "androidx.test.espresso:espresso-core:3.4.0"
+    androidTestImplementation "androidx.test.espresso:espresso-contrib:3.4.0"
+    androidTestImplementation "androidx.test.espresso:espresso-intents:3.4.0"
+    androidTestImplementation "androidx.test.espresso:espresso-accessibility:3.4.0"
+    androidTestImplementation "androidx.test.espresso:espresso-web:3.4.0"
+    androidTestImplementation "androidx.test.espresso.idling:idling-concurrent:3.4.0"
+
+//    androidTestImplementation "androidx.test.espresso:espresso-idling-resource:$espressoVersion"
```
Comment (P1200) [inline app/build.gradle:68]: Commented out code.

File: app/src/main/res/layout/activity_create_event.xml [hunk 7]
```diff
@@ -1,13 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 
+
```
Comment (P1200) [inline app/src/main/res/layout/activity_create_event.xml:3]: Redundant line.

**Review by P1200 — APPROVED** [review]

**Pull request merged**